README.md

```diff
@@ -66,7 +66,6 @@ The output is going to look like the following:
 [+] Counting how many people contributed to the project in the last three months ...
 [+] Counting how many stars the project has ...
 [+] Counting how many watchers the project has ...
-[+] Figuring out when the first commit was done ...
```

Comment:
Please update this README.md with the latest changes

Reply:
The sample output has to be re-created. I did it but looks like my GitHub token hits the rate limit. As a result, the tool can't figure out if the project uses signed commit. I think it's fine. The output contains all features and scores - that should be enough.

Reply:
Yes it contains all the features.